testsuite/mlir/mlir-parse/dialect.mlir

```diff
@@ -23,38 +23,40 @@
 // }
 // ```
 
+!C = type !verona.class<"C">
+!D = type !verona.class<"D", "f" : meet<U64, imm>, "g" : meet<!C, mut>>
 module {
   verona.class @C {
   }
 
   verona.class @D {
     verona.field "f" : !verona.meet<U64, imm>
-    verona.field "g" : !verona.meet<class<"C">, mut>
+    verona.field "g" : !verona.meet<!C, mut>
   }
 
   func @bar(%x: !verona.meet<U64, imm>, %y: !verona.meet<U64, imm>) {
-    %a = verona.new_region @C [ ] : !verona.meet<class<"C">, iso>
-    %b = verona.view %a : !verona.meet<class<"C">, iso> -> !verona.meet<class<"C">, mut>
+    %a = verona.new_region @C [ ] : !verona.meet<!C, iso>
+    %b = verona.view %a : !verona.meet<!C, iso> -> !verona.meet<!C, mut>
```

Comment:
This is **so** much nicer than having to redefine `class<"C">` and `class<"D">` everywhere.